pg_search/tests/pg_regress/sql/unified_expression_comprehensive.sql

```diff
@@ -623,8 +623,8 @@ SELECT
 FROM products
 WHERE
   (products.id @@@ paradedb.all())
-  AND (products.name ILIKE ANY (array['%Socks%']))
-  AND (products.created_at < (SELECT created_at FROM products WHERE products.id = 1978) OR products.id < 1978 AND products.created_at = (SELECT created_at FROM products WHERE products.id = 1978))
+  AND (products.name ILIKE ANY (array['%Nike%', '%Adidas%']))
+  AND (products.created_at < (SELECT created_at FROM products WHERE products.id = 8) OR products.id < 8 AND products.created_at = (SELECT created_at FROM products WHERE products.id = 8))
 ORDER BY products.created_at DESC, products.id DESC
 LIMIT 100;
```

Comment:
The goal of this test was to not match any results in the subquery, hence the `id = 1978`: in one case it reproduced a separate error like:
```
PostgresExpression should not evaluate to NULL
```

See the ticket for more info.